editoast/editoast_schemas/src/rolling_stock.rs

```diff
@@ -54,14 +55,14 @@ pub struct RollingStock {
     pub effort_curves: EffortCurves,
     pub base_power_class: Option<String>,
     /// In m
-    pub length: f64,
+    pub length: Length,
     /// In m/s
-    pub max_speed: f64,
-    pub startup_time: f64,
+    pub max_speed: Velocity,
```

Comment:
I would have expected this type to contain the unity, but it's not. If I understand, it would be possible to do, but that would be against the default generic which seems to be System International (meter, kilograms, seconds, etc.). Not sure what are your opinions about that? If I got it right, in any case, the type system will protect us against misuse of unities anyway...